### ISSUE TITLE
Doc history doesn't display correctly

### ISSUE THREAD

**Issue by bree-z** [cuny-academic-commons/commons-in-a-box#160]
Hi Boone -- I was able to reproduce a user-reported bug, where the history for docs doesn't display correctly.  It just says "You are currently viewing a revision titled "", saved on  by"

See attached screenshots for the history on this doc: https://openlabdev.commonsinabox.org/groups/open-pedagogy-on-the-openlab/docs/open-pedagogy/history/  and compared to a doc on the OpenLab: https://openlab.citytech.cuny.edu/groups/research-project/docs/history/history/

![dochistory_cboxol](https://user-images.githubusercontent.com/7189824/49546970-85a5c900-f8af-11e8-8216-9bc92f8a99b7.png)

![dochistory_ol](https://user-images.githubusercontent.com/7189824/49546984-89d1e680-f8af-11e8-8d54-7a1c2ef24000.png)

**Comment by boonebgorges** [comment]
Thanks for this. It was probably missed during pre-release QA. The City Tech OpenLab runs a very outdated copy of BuddyPress Docs, and the template was probably not updated for the more modern version when everything was ported to CBOX.

**Comment by bree-z** [comment]
Hi @boonebgorges - just wanted to check in about this one.  Is it in a future release because it's too complicated for the 1.1.1 milestone?  

Thanks!

**Comment by boonebgorges** [comment]
Yes. It is a large task, and is outside the scope of what we're able to do with our current maintenance resources.

**Comment by bree-z** [comment]
Got it.  Thanks Boone!
